texastribune/donations
# Remove ambassador page

#### What's this PR do?

Removes the ambassador page (`/account/ambassador/`) in the portal

#### Why are we doing this? How does it help us?

There is some underlying tech debt with how we generate ambassador links. This helps us address that. Only about 5% of portal traffic visits this page

#### How should this be manually tested?

The main thing here is to make sure nothing breaks with all these deleted pieces. I'll throw this up on staging so that we can run through the main steps: donating and clicking around in the portal.

I think ambassador, twitter, etc. links get created in the portal api when a contact is created so it makes sense to create a fresh user for the donations testing portion.

#### How should this change be communicated to end users?

The proper teams know about this

#### Are there any smells or added technical debt to note?

Let me know if I missed any lingering ambassador logic. I did leave [this query param logic](https://github.com/texastribune/donations/pull/284) in there. I figure that might be useful if we created ambassador links on the fly (without yourls) someday.

#### What are the relevant tickets?

https://airtable.com/appyo1zuQd8f4hBVx/tbloNZu8GkM52NKFR/viwb2L2CHc4ZWqkC5/reccBNvVg0jicsluB?blocks=hide

#### Have you done the following, if applicable:
***(optional: add explanation between parentheses)***


#### TODOs / next steps:

* [ ] *Remove the link generation in the portal*
